src/bp-core/compatibility/bp-incompatible-plugins-helper.php

```diff
@@ -43,6 +43,15 @@ function bp_helper_plugins_loaded_callback() {
 		require buddypress()->compatibility_dir . '/bp-rankmath-plugin-helpers.php';
 	}
 
+	/**
+	 * Include plugin when plugin is activated
+	 *
+	 * Support Rank Math SEO
```

Comment:
@mahedihasannoman Is this related to Rank Math SEO?

Reply:
Changed.